solutions/scripts/scripts.js

```diff
@@ -37,7 +43,10 @@ window.hlx.plugins.add('rum-conversion', {
 window.hlx.plugins.add('experimentation', {
   condition: () => getMetadata('experiment'),
   options: {
+    audiences: AUDIENCES,
```

Comment:
@andreibogdan I wonder if "audiences" param is necessary. It might introduce an unintended dimension to the experiment, and we also don't include it in the report.

Reply:
I added this for testing purposes. I will remove it.